Makefile

```diff
@@ -30,13 +30,11 @@ LANTERN_BINARIES_PATH ?= ../lantern-binaries
 
 GO_MOBILE_REVISION=47553f4d4275316f2610b3204f8f9ecaf3ce63a7
 
-GH_USER := getlantern
```

Comment:
A bit of cleaning now that I'm here. Removing commented out value and using a conditional declaration.